doc/api_rstgen.py

```diff
@@ -1,8 +1,87 @@
 """Provides a module for generating PyFluent API RST files."""
 
+import os
 from pathlib import Path
 import shutil
 
+api_contents_path = (
+    Path(__file__).parents[0].resolve() / "source" / "api" / "api_contents.rst"
+)
+fluent_version = "25.2"
```

Comment:
Can we use `FluentVersion.current_release()` from PyFluent?

Reply:
Yes, right.

Reply:
Done. Thank you.